int/api/cmd/execute_function.go

```diff
@@ -13,40 +11,37 @@ import (
 	"github.com/massalabs/thyra/pkg/wallet"
 )
 
-func NewExecuteFunction(walletStorage *sync.Map) *FunctionExecuter {
-	return &FunctionExecuter{walletStorage: walletStorage}
+func NewExecuteFunction() *FunctionExecuter {
+	return &FunctionExecuter{}
 }
 
-type FunctionExecuter struct {
-	walletStorage *sync.Map
-}
+type FunctionExecuter struct{}
 
 //nolint:nolintlint,ireturn
 func (f *FunctionExecuter) Handle(params operations.CmdExecuteFunctionParams) middleware.Responder {
```

Comment:
Without the constructor, should be something like:

```suggestion
func ExecuteFunctionHandler(params operations.CmdExecuteFunctionParams) middleware.Responder 
```